libfreerdp/core/freerdp.c

```diff
@@ -705,6 +705,8 @@ BOOL freerdp_context_new(freerdp* instance)
 	update_register_client_callbacks(rdp->update);
 	instance->context->abortEvent = CreateEvent(NULL, TRUE, FALSE, NULL);
 
+	update_register_io_callbacks(rdp->update);
```

Comment:
Wrap this in a public `freerdp_register_transport_callbacks` or similar. Set default functions to the ones already reading/writing